### PR TITLE
Language Manual Banner Component For Learn Area Redesign

### PR DESCRIPTION
**Pull request by SaySayo**
Built a component for the language manual banner according to the learn area redesign. cc @sabine

Figma design
figma design

Created component
created component

### CODE REVIEW

File: src/ocamlorg_frontend/components/learn_overview.eml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+let lang_manual_banner =
+  <div class="container bg-learn-area-orange w-full h-80 flex items-center px-4">
+    <div class="w-2/3 pr-4 mb-2">
+      <h2 class="mt-0 mb-2 text-white text-sm font-medium leading-7 tracking-widest">GUIDE</h2>
+      <h3 class="mt-0 mb-4 text-white text-2xl font-normal leading-10">Language Manual</h3>
+      <p class="mt-0 mb-4 text-white text-lg font-normal leading-7">OCaml language manuals are comprehensive guides covering syntax, features, and usage.
+        They assist developers and learners in understanding capabilities, best practices, and exploring functionalities.</p>
```
Comment (sabine) [inline src/ocamlorg_frontend/components/learn_overview.eml]:
```suggestion
      <p class="mt-0 mb-4 text-white text-lg font-normal leading-7">The OCaml Language Manual is a comprehensive guide covering syntax, features, and usage.
        The OCaml compiler developers provide this resource to assist developers and learners in understanding capabilities, best practices, and exploring functionalities.</p>
```

File: src/ocamlorg_frontend/components/learn_overview.eml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+let lang_manual_banner =
+  <div class="container bg-learn-area-orange w-full h-80 flex items-center px-4">
+    <div class="w-2/3 pr-4 mb-2">
```
Comment (sabine) [inline src/ocamlorg_frontend/components/learn_overview.eml]: container needs to be inside the surrounding content, and don't use `container`, but `container-fluid` is probably more appropriate here.

**Review by sabine — COMMENTED** [review]
since this is a banner, it probably makes sense to have it be fully clickable

Collapsing on small screens: Image stacked on top of the rest of the content.

**Comment by tmattio** [comment]
Thanks @SaySayo!

**Comment by SaySayo** [comment]
Hi @tmattio, this wasn't meant to be merged yet as there's a bit of a bug in the responsiveness on smaller screens. I should've marked it as a draft. My apologies about the inconvenience. Will send a PR fixing the bug ASAP!

**Pull request merged**